src/main/java/org/opendevstack/provision/authentication/crowd/CrowdSecurityConfiguration.java

```diff
@@ -232,11 +222,42 @@ public void onAuthenticationSuccess(
     return successHandler;
   }
 
-  /**
-   * define an failure handler in case of an authentication failure
-   *
-   * @return
-   */
+  @NotNull
+  public SimpleUrlAuthenticationSuccessHandler createAuthSuccessHandlerThatReturnsOK() {
+    SimpleUrlAuthenticationSuccessHandler handler =
+        new SimpleUrlAuthenticationSuccessHandler() {
```

Comment:
I sense that this could be expressed as a lambda expression

Reply:
yes, it could :-)